modules/_labs/select/react/stories/stories_testing.tsx

```diff
@@ -0,0 +1,136 @@
+/// <reference path="../../../../../typings.d.ts" />
+import * as React from 'react';
+
+import {colors} from '@workday/canvas-kit-react-core';
+import {Button} from '../../../../button/react';
+import FormField from '../../../../form-field/react';
+import Modal from '../../../../modal/react';
+import Select from '../lib/Select';
+
+import {manyOptions, options} from './stories';
+import {controlComponent} from '../../../../../utils/storybook';
+
+export default {
+  title: 'Testing|React/Labs/Select',
+  component: Select,
+};
+
+const Container = ({children, style = {}, ...elemProps}) => {
+  return (
+    <div
+      style={{
+        border: `1px solid ${colors.soap600}`,
+        marginTop: 50,
+        overflow: 'hidden',
+        padding: 20,
+        ...style,
+      }}
+      {...elemProps}
+    >
+      {children}
+    </div>
+  );
+};
+
+const SelectModal = () => {
+  const [open, setOpen] = React.useState(false);
```

Comment:
There's a `useModal` hook for just such an occasion: https://github.com/Workday/canvas-kit/blob/53b7f49e96540850c48c90465fe7ae9498e6953c/modules/modal/react/stories/stories.tsx#L20-L37

Reply:
Used the `useModal` hook.